src/main/resources/story/web_app/Trello End To End Scenarios.story

```diff
@@ -0,0 +1,90 @@
+|active-board           |By.xpath(//a[@aria-label='Board with tasks (currently active)'])                             |
+|invalid-password       |Lu9iomjlp0                                                                                   |
+
+
+Scenario: Verify login to Trello
+Given I am on page with URL `https://trello.com/`
+When I click on element located by `<login-button>`
+When I enter `<email>` in field located by `<email-field>`
+When I click on element located by `<continue-button>`
+When I wait until element located by `<password-field>` appears
+When I enter `<password>` in field located by `<password-field>`
+When I click on element located by `<login-submit-button>`
+Then the page with the URL containing '/svitlanasomina/' is loaded
+
+
+Scenario: Verify creating the board
+When I click on element located by `<create-new-board-button>`
+When I enter `Board with tasks` in field located by `<board-title-field>`
+When I click on element located by `<create-button>`
+When I wait until element located by `<active-board>` appears
+Then number of elements found by `<active-board>` is equal to `1`
+
+
+Scenario: Verify creating the list
+When I wait until element located by `<list-name-input>` appears
+When I enter `new list` in field located by `<list-name-input>`
+When I click on element located by `<list-add-button>`
+Then number of elements found by `By.xpath(//div[@data-testid='list-header'])` is equal to `1`
+
+
+Scenario: Verify adding cards
+When I click on element located by `<add-card>`
+When I enter `#{generate(numerify 'card####')}` in field located by `<card-title-field>`
+When I click on element located by `<add-card-button>`
+When I enter `#{generate(numerify 'card####')}` in field located by `<card-title-field>`
+When I click on element located by `<add-card-button>`
+When I change context to element located `By.xpath((//div[@class='list js-list-content'])[1])`
+When I ESTABLISH baseline with name `new_list`
+When I COMPARE_AGAINST baseline with name `new_list` ignoring:
+|ACCEPTABLE_DIFF_PERCENTAGE|ELEMENT                                               |
+|30                        |By.xpath(//div[@class='list-card-details u-clearfix'])|
+When I reset context
+
+
+Scenario: Verify closing the board
+When I click on element located by `<active-board>`
+When I click on element located by `By.xpath((//button[@aria-label='Board actions menu'])[1])`
+When I wait until element located by `By.xpath(//button[@title='Close board'])` appears
+When I click on element located by `By.xpath(//button[@title='Close board'])`
+When I click on element located by `By.xpath(//button[@title='Close'])`
+When I wait until element located by `By.xpath(//h1[text()='Board with tasks is closed.'])` appears
+Then number of elements found by `By.xpath(//h1[text()='Board with tasks is closed.'])` is equal to `1`
+
+
+Scenario: Verify logout from Trello
```

Comment:
[improvement suggestion] I would move login and logout scenarios to composite steps.